examples/asr/conf/config.yaml

```diff
@@ -1,4 +1,4 @@
-#model: "QuartzNet15x5"
+name: &name "QuartzNet15x5"
```

Comment:
YAML anchors are not good enough for Hydra because they are resolved at load time and config composition happens at run time.
You should use OmegaConf interpolations instead:

```yaml
name: QuartzNet15x5

exp_manager:
  root_dir: null
  name: ${name}
```

On a side note, Hydra has [`hydra.job.name`](https://hydra.cc/docs/next/configure_hydra/job) which is probably exactly the same. you should be using that.